packages/nextra/src/loader.ts

```diff
@@ -110,6 +117,15 @@ async function loader(
     },
     resourcePath
   )
+
+  // Imported as a normal component, no need to add the layout.
+  if (!__nextra_page_import__) {
+    return `
+${themeIncludeStyles && `import '${theme}/style.css'`}
```

Comment:
Because it’s a boolean and can be evaluated to `false`
```suggestion
${themeIncludeStyles ? `import '${theme}/style.css'` :  ''}
```